app/views/devise/passwords/edit.html.erb

```diff
@@ -1,25 +1,42 @@
-<h2>Change your password</h2>
-
-<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
-  <%= render "devise/shared/error_messages", resource: resource %>
-  <%= f.hidden_field :reset_password_token %>
-
-  <div class="field">
-    <%= f.label :password, "New password" %><br />
-    <% if @minimum_password_length %>
-      <em>(<%= @minimum_password_length %> characters minimum)</em><br />
-    <% end %>
-    <%= f.password_field :password, autofocus: true, autocomplete: "new-password" %>
+<div class="row fullscreen">
+  <div class="col-12">
+    <div class="logoline">Get Together with <%= image_tag( 'logo.svg', alt: 'GatherPack', class: 'logo') %> GatherPack</a></div>
   </div>
+  <div class="col-md-6">
+    <div class="card">
+      <div class="card-header">
+        <h2>Change your password</h2>
+      </div>
+      <div class="card-body">
+        <%= simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
+          <%= render "devise/shared/error_messages", resource: resource %>
+          <%= f.hidden_field :reset_password_token %>
 
-  <div class="field">
-    <%= f.label :password_confirmation, "Confirm new password" %><br />
-    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
-  </div>
+          <div class="field">
+            <%= f.input :password, autofocus: true, autocomplete: "new-password", label: "New password" %>
+            <% if @minimum_password_length %>
+              <em>(<%= @minimum_password_length %> characters minimum)</em><br />
+            <% end %>
+          </div>
 
-  <div class="actions">
-    <%= f.submit "Change my password" %>
-  </div>
-<% end %>
+          <div class="field">
+            <%= f.input :password_confirmation, autocomplete: "new-password", label: "Confirm new password" %>
+          </div>
 
-<%= render "devise/shared/links" %>
+          <div class="actions">
+            <%= f.submit "Change my password", class: 'btn btn-primary' %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    <div class="card">
+      <div class="card-header">
+        <h2>Create an Account with Email &amp; Password</h2>
+      </div>
+      <div class="card-body">
+        <%= link_to 'Sign Up', new_registration_path(resource_name), class: 'btn btn-primary' %>
+      </div>
+    </div>
```

Comment:
If you're resetting your password, you probably don't want the option to create a new account instead...

Reply:
Should I remove it from the confirmations and unlocks as well?

Reply:
I have removed it from the reset password screen.